terraform/README.md

```diff
@@ -0,0 +1,49 @@
+# Deploy Online Boutique sample application on a GKE cluster
+
+This page walks you through the steps required to deploy the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) sample application on a GKE ([Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)) cluster using Terraform.
+
+## Pre-requisites
+
+Setting up the sample requires that you have a [Google Cloud Platform (GCP) project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#console), connected to your billing account.
+
+## Deploying the sample application
+
+Once you have ensured that all the pre-requisites are met, follow the steps below to create a GKE cluster and deploy the Online Boutique sample application:
+
+1. Clone the repo:
+    ```
+    git clone https://github.com/GoogleCloudPlatform/microservices-demo.git
+    ```
+1. Move into the Terraform directory that has the installation scripts:
+    ```
+    cd microservices-demo/terraform
+    ```
+1. Navigate into the *variables.tf* file and set the gcp_project_id variable to your Google Cloud project ID: `default = "<project_id_here>"`
```

Comment:
My understanding is that variables.tf file should contain variable declarations necessary for the installation, and optionally a default value. If a user-supplied overrides to a default values is desired, it should be done in a `terraform.tfvars` file, which gets automatically read by terraform. I would suggest having a similar setup here. See this PR for reference:

https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/881/files#diff-27eed1e914ea8c9ef555a47de926c6e96275c461a0ad124606dfd25eefc156f0

Reply:
Sounds good! This aligns with what @ckim328 has suggested so I will look to alter the implementation.